### PR TITLE
build: init sbom action

### PR DESCRIPTION
**Pull request by hblitza**
This adds an action to create a SBOM for this project.

### CODE REVIEW

File: .github/workflows/on-push-main.yml [hunk 1]
```diff
@@ -90,3 +92,14 @@ jobs:
...
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npx --package=node@v18-lts -- "npx semantic-release"
+
+      - name: Generate JSON SBOM 🏛️
+        uses: CycloneDX/gh-node-module-generatebom@v1
+        with:
+          output: sbom.json
+
+      - name: Commit SBOM 👈
+        uses: EndBug/add-and-commit@v9
```
Comment (simonseyock) [inline .github/workflows/on-push-main.yml:102]: fyi: If action/checkout is used, you don't need an external action to commit something:
https://stackoverflow.com/questions/57921401/push-to-origin-from-github-action/58393457#58393457

Comment (hblitza) [reply .github/workflows/on-push-main.yml]: ok thanks! please check again.

**Review by simonseyock — APPROVED** [review]
nice!

**Comment by hblitza** [comment]
thx for review

**Pull request merged**